resources/lib/api/graphql.py

```diff
@@ -155,6 +158,7 @@ def __get_latest_for_genre(self, genre):
       info = {
         "duration": item["duration"]
       }
+      season_title = ""
```

Comment:
Does this run? `season_title` should be passed to the VideoItem constructor...

Reply:
Oops, removed